.flake8

```diff
@@ -0,0 +1,4 @@
+[flake8]
+exclude = docs, scanpy/tests
+max-line-length = 120
+ignore = F401, W503, E501, E203, E231, W504, E402, E126, E712, E741
```

Comment:
For checks that are switched off, could there be some documentation about why? If it's taken from a template it'd be fine to just link to where that came from.